Framework/Algorithms/src/Integration.cpp

```diff
@@ -380,7 +380,7 @@ MatrixWorkspace_sptr Integration::getOutputWorkspace(MatrixWorkspace_sptr inWS,
   if (inWS->id() == "RebinnedOutput") {
     MatrixWorkspace_sptr outWS = API::WorkspaceFactory::Instance().create(
         "Workspace2D", maxSpec - minSpec + 1, 2, 1);
-    API::WorkspaceFactory::Instance().initializeFromParent(*inWS, *outWS, true);
+    API::WorkspaceFactory::Instance().initializeFromParent(*inWS, *outWS);
```

Comment:
If `inWs` has `blocksize() == 1` this will now copy bin masks, whereas the old code did not, I think?